forms/jqm/element.js

```diff
@@ -68,9 +71,13 @@ define(function (require) {
     renderLabel: function () {
       if (!this.$label) {
         this.$label = $('<label class="ui-input-text"></label>');
+      }
+
+      if (!this.$el.find(this.$label).length) {
         this.$el.append(this.$label);
       }
-      this.$label.html(this.model.attributes.label || '');
+
+      this.$label.text(this.model.attributes.label || '');
```

Comment:
ok changed it back.

There are 20 other places in forms that use .text to set things so we may want to review them when someone has some spare time